andersen-homework/src/main/java/org/util/CustomArrayList.java

```diff
@@ -0,0 +1,47 @@
+package org.util;
+
+import java.util.Arrays;
+
+public class CustomArrayList <T>{
+    private Object[] items;
+    int size=0;
```

Comment:
What this field has default access modifier?

Reply:
int size has a default access modifier.It makes field accessible within the same package